### PR TITLE
Updating/Fixing several CO sources

### PR DESCRIPTION
**Pull request by slibby**
REST service went missing, found zipfile

### CODE REVIEW

**Comment by ingalls** [comment]
This is fantastic! Glad to have another set of eyes on CO!

File: sources/us-co-sanmiguel.json [hunk 1]
```diff
@@ -24,4 +24,4 @@
         ],
```
Comment (ingalls) [inline sources/us-co-sanmiguel.json]: Looks like oa/hooked failed as one of these fields doesn't exist

**Comment by slibby** [comment]
looks like this now conflicts after all the files were moved to subdirectories -- not  sure if it's easier for a repo owner to fix or to cancel the pr and re-add.

**Comment by ingalls** [comment]
@slibby I'll fix these!

**Comment by ingalls** [comment]
All in, thanks!

**Pull request closed without merging**